src/content/en/2024/javascript.md

```diff
@@ -1,19 +1,809 @@
 ---
+
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: JavaScript
 description: JavaScript chapter of the 2024 Web Almanac covering the usage of JavaScript on the web, libraries and frameworks, compression, web components, and source maps.
 hero_alt: Hero image of the Web Almanac characters cycling to power a website.
-authors: []
-reviewers: []
-editors: []
-analysts: []
+authors: [haddiamjad]
+reviewers: [tunetheweb]
+editors: [tunetheweb]
+analysts: [nrllh]
 translators: []
 results: https://docs.google.com/spreadsheets/d/16isMe5_rvmRmJHtK5Je66AhwO8SowGgq0EFqXyjEXw8/
-featured_quote:
-featured_stat_1:
-featured_stat_label_1:
-featured_stat_2:
-featured_stat_label_2:
-featured_stat_3:
-featured_stat_label_3:
```

Comment:
@hadiamjad please confirm you're happy with these quotes.

You can see what these look like here:
https://javascript-2024-dot-webalmanac.uk.r.appspot.com/en/2024/?feat=javascript#featured-chapter